pyproject.toml

```diff
@@ -614,6 +615,7 @@ spaces_indent_inline_array = 4  # Match Python PEP 8
 trailing_comma_inline_array = true
 
 [tool.uv.sources]
+"aviary.cloning" = {workspace = true}
```

Comment:
In `tool.mypy`'s `mypy_path`, can you append `$MYPY_CONFIG_FILE_DIR/packages/cloning` to it